apps/contrib/templates/a4_candy_contrib/includes/form_field.html

```diff
@@ -8,15 +8,21 @@
     </label>
     {% block help_text %}
     {% if field.help_text %}
-    <div class="form-hint">
+    <p class="form-hint" id="hint_{{ field.id_for_label }}">
         {{ field.help_text }}
-    </div>
+    </p>
     {% endif %}
     {% endblock %}
     {% block field %}
         <div class="widget widget--{{ field|widget_type }}">
+          {% if field.errors %}
+            {% render_field field|attr:"aria-invalid=true"|attr:"aria-describedby=hint_{{field.id_for_label}}" %}
+          {% else %}
```

Comment:
I would sooooooo love this to work (`aria-describedby`) but sadly it doesn't, I have been faffing for too long though so will see if you guys have any suggestions and if not will remove

Reply:
@phillimorland I think this works, maybe you can double check?

Reply:
alternatively we can add a templatetag I think

Reply:
oooo nice, I tried with `with``but didn't use/find out about `|add`, will check it :)